apps/aefate/src/aefa_fate.erl

```diff
@@ -260,7 +260,16 @@ abort(E) -> throw({add_engine_state, E}).
 
 execute(EngineState) ->
     Instructions = aefa_engine_state:current_bb_instructions(EngineState),
-    loop(Instructions, EngineState).
```

Comment:
If debugging is enabled via macro, engine state can just assume the debugger is in

Reply:
This will do 79686760aa7bbb12c21ef7d4026f4eab0743ec0f ?

Reply:
Yes